src/test/java/lotto/domain/MoneyTest.java

```diff
@@ -0,0 +1,55 @@
+package lotto.domain;
+
+import org.junit.jupiter.api.Test;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+public class MoneyTest {
+    private static final String MONEY_UNIT_ERROR = "[ERROR] 금액을 1000단위로 입력해주세요";
+    private static final String MONEY_INT_ERROR = "[ERROR] 숫자만 입력할 수 있습니다";
+    private static final String MONEY_RANGE_ERROR = "[ERROR] 금액을 1000원 이상 입력해주세요";
+
+    @Test
+    void 구입금액이_숫자인지_확인() {
+        String falseMoney = "로또값";
+        assertThatThrownBy(() -> {
+            new Money(falseMoney);
+        }).isInstanceOf(IllegalArgumentException.class)
+                .hasMessageContaining(MONEY_INT_ERROR);
```

Comment:
이렇게도 사용할 수 있을 것 같아요. 
```suggestion
                .hasMessageContaining(Money.MONEY_INT_ERROR);
```